python/llm/dev/benchmark/all-in-one/run-deepspeed-arc.sh

```diff
@@ -10,7 +10,11 @@ source $basekit_root/ccl/latest/env/vars.sh --force
 
 NUM_GPUS=2 # number of used GPU
 export USE_XETLA=OFF
-export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=2
+if grep -q "Xeon" /proc/cpuinfo; then
+    export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=0
```

Comment:
Can we just don't set it in this case?

Reply:
> Can we just don't set it in this case?

Have changed to only set it to be 2 when Core CPU.